lib/cartopy/tests/test_img_nest.py

```diff
@@ -191,7 +191,6 @@ def world_file_extent(*args, **kwargs):
         return extent, pix_size
 
 
-@pytest.mark.xfail(reason='MapQuest is unavailable')
```

Comment:
I think this is the test failing on AppVeyor?

Reply:
Ahh, yes, thanks for catching that! I don't think this is the right xfail. I think it was failing on all Windows installs due to the path separator in the path. Just pushed a new commit to see if that does it.